index.d.ts

```diff
@@ -0,0 +1,18 @@
+declare module "asyncify-wasm" {
```

Comment:
Instead of this, a better practice is to define module-like definitions, so that they're not tied up to a global name.

That is, unwrap the `declare module` and instead use `export` on individual classes / functions.